web-frontend/src/pages/TransferCoins/ReceiveCoins/GenerateLink.tsx

```diff
@@ -13,6 +13,12 @@ import {
   PopupModalHeader,
   Selector,
 } from '@massalabs/react-ui-kit';
+import { validateInputs } from '../../../validation/sendInputs';
+
+interface IError {
```

Comment:
rename this interface to something like SendInputErrors.

declare this interface in ../../../validation/sendInputs and import it here, modify the return type of validateInputs to this interface.